src/tagmanager/tm_tag.h

```diff
@@ -154,7 +154,7 @@ gboolean tm_tag_is_anon(const TMTag *tag);
 
 const char *tm_tag_type_name(const TMTag *tag);
 
-TMTagType tm_tag_name_type(const char* tag_name);
+TMTagType tm_tag_name_type(const char* tag_name, TMParserType lang);
```

Comment:
can't find the corresponding change in the C file-

Reply:
No change, its fixing the .h to match the existing .c

Reply:
Reference https://github.com/geany/geany/blob/987cc3f106bf1eb6c0c0667bf5f24582022180ae/src/tagmanager/tm_tag.c#L692

Reply:
Yes.

This change is quite independent from this PR, I just needed to define TM_DEBUG for debugging this PR so this is why it's here.